src/font.c

```diff
@@ -453,6 +453,10 @@ gdip_font_clear_pattern_cache (void)
 static GpStatus
 create_fontfamily_from_collection (char* name, GpFontCollection *font_collection, GpFontFamily **fontFamily)
 {
+	/* Initialize the font set. */
+	if (font_collection->config)
+		gdip_createPrivateFontSet (font_collection);
```

Comment:
I wonder if we should move this up to `GdipCreateFontFamilyFromName()` instead? It'd be more consistent with  `GdipGetFontCollectionFamilyCount()` and `GdipGetFontCollectionFamilyList()`.